### PR TITLE
feat: Slider for adjusting contrast and brightness

### LINKED ISSUE #30: Brightness and Contrast
**Is your feature request related to a problem? Please describe.**

We need to have brightness and contrast control for the background canvas.

**Describe the solution you'd like**
Buttons and/or sliders to control the canvas filters for brightness and contrast.

**Describe alternatives you've considered**
N/A

**Additional context**
Should also update the minimap.

### PR DESCRIPTION
**Pull request by SilviaZeta**
# Description

Adds sliders for adjusting contrast and brightness, as described in #30

I've added the sliders to the UI as part of a new "Background" accordion. Do we prefer to have them as buttons in a toolbar on the left side of the canvas?

Screenshot 2021-03-23 at 13 48 34

closes #30

# Dependency changes

No

# Testing

N/A

# Documentation

N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] ~I have made corresponding changes to the documentation~
- [x] My changes generate no new warnings
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [ ] ~New and existing unit tests pass locally with my changes~

### CODE REVIEW

**Review by ChrisBaidoo — DISMISSED** [review]
LGTM

File: src/components/BaseSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+export enum Sliders {
+  contrast,
+  brightness,
+}
+
+export const sliderConfig: {
+  [id: number]: SliderConfigItem;
+} = {
+  [Sliders.contrast]: {
```
Comment (ChasNelson1990) [inline src/components/BaseSlider.tsx:12]: Genuine question - is it really a base slider if it has specific configs for sliders in?

File: src/components/BaseSlider.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+export const BaseSlider: FunctionComponent<Props> = ({
+  value,
+  slider,
+  onChange,
+}): ReactElement => {
+  const config = sliderConfig[slider];
+  return (
+    <>
+      <Typography
+        id="continuous-slider"
```
Comment (ChasNelson1990) [inline src/components/BaseSlider.tsx:49]: I don't think the id should be set by the base component, ids should be unique and they won't be here - all sliders will share the same id... which means it's actually a class...

File: src/components/BaseSlider.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        gutterBottom
+        style={{ textTransform: "capitalize" }}
+      >
+        {config.name}
+      </Typography>
+      <Slider
```
Comment (ChasNelson1990) [inline src/components/BaseSlider.tsx:55]: Could we add some custom marks based on the min max and default? E.g. https://material-ui.com/components/slider/#custom-marks

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

File: src/components/BaseSlider.tsx (hunk 1 above)
Comment (SilviaZeta) [reply src/components/BaseSlider.tsx:12]: No! I will move any config out and make it a real base slider.

**Review by ChrisBaidoo — APPROVED** [review]
LGTM

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**